### PR TITLE
add include support to analysis_config.yaml file. 

### PR DESCRIPTION
**Pull request by rubensantoscodacy**
Supported includes atm [lints](https://pub.dev/packages/lints
) and [flutter_lints](https://pub.dev/packages/flutter_lints)

### CODE REVIEW

**Review by lucassklp — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -8,69 +8,15 @@ This is the docker engine we use at Codacy to have [Dart Analyzer](https://githu
 You can also create a docker to integrate the tool and language of your choice!
 See the [codacy-engine-scala-seed](https://github.com/codacy/codacy-engine-scala-seed) repository for more information.
 
-## Limitations
+##Supported packages lints on analysis_config.yaml files
```
Comment (lolgab) [inline README.md:11]:
```suggestion
## Supported packages lints on analysis_config.yaml files
```

File: doc-generator/lib/generatedocs.dart [hunk 2]
```diff
@@ -293,3 +296,106 @@ abstract class ErrorCodeInfo {
...
+    dartanalyzerDir.createSync(recursive: true);
+  }
+
+  /*
+  name: "dartanalyzer"
+  version: "0.0.1"
+  description: "Initial pubspec.yaml to install supported includes on analysis_options.yaml files"
+  environment:
+    sdk: "2.16.1"
+   */
```
Comment (lolgab) [inline doc-generator/lib/generatedocs.dart:318]: Is this comment here to stay? Can you elaborate on what is this?

File: doc-generator/lib/generatedocs.dart [hunk 3]
```diff
@@ -293,3 +296,106 @@ abstract class ErrorCodeInfo {
...
+  YamlWriter({
+    this.spaces = 2,
+  });
+
+  /// Write a dart structure to a YAML string. [yaml] should be a [Map] or [List].
+  String write(dynamic yaml) {
+    return _writeInternal(yaml).trim();
+  }
+
+  /// Write a dart structure to a YAML string. [yaml] should be a [Map] or [List].
```
Comment (lolgab) [inline doc-generator/lib/generatedocs.dart:352]: Aren't libraries out there to serialize to YAML?

**Review by lolgab — CHANGES_REQUESTED** [review]
Some minor comments, otherwise LGTM! :)

**Review by lolgab — APPROVED** [review]

**Pull request merged**